xcell/mappers/mapper_KV450.py

```diff
@@ -186,7 +185,7 @@ def get_mask_mod():
                                       rot=self.rot)
             return msk
 
-        fn = '_'.join([f'KV450_mask_{kind}_bin{self.zbin}',
+        fn = '_'.join([f'mask_{self.mask_name}_{kind}',
```

Comment:
Why do you keep this here for KV450?

Reply:
KiDS and KV450 can generate masks not just for the catalog but for the star subsample.
If you want them to adhere to the rest of the mappers, we would need to drop that functionality.

Reply:
But not in the same run, right? If you want to keep this interactive functionality, you could add `**kwargs` to `get_mask` and pass them `_rerun_read_cycle` and to `_get_mask`.

Reply:
you could generate them in the same run. You just need to add two mappers with different `mode` values in the config dictionary.

I think it's not worth the complication. To some extent this is what overriding is for

Reply:
... unless we want to assume that users (again, us) would be attentive enough to pass different mask names to both yaml sections... If you're happy to assume that (at the end of the day we are assuming that for all other mappers), then this could be incorporated into the general structure.

Reply:
Yes. I would rather unify things. And as you say, it is reasonable to assume the user will choose the right mask name.

Reply:
OK, done. Take a look!